<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChambreAdmin</class>
 <widget class="QDialog" name="ChambreAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
    selection-background-color:red;
border-radius:15px;
}

QDialog#ChambreAdmin
{background-image:url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}
 
QLabel
{
color:white;
background:transparent;

}

QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }


QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton#ajouterchambre_btn:hover
{

color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}

QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>951</width>
     <height>621</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableViewchdispoadmin">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>341</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterchambre_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ajouter chambre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chambres disponibles :</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewchambreadmin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>341</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir le numéro de chambre à chercher  :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cherchambreadmin_txt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
